example/vanilla-k8s-dashboard/CSI-CNS-Dashboard-v1.0.json

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
I have two questions:
1. Does it make sense to add dashboard for CNS operations in CSI driver? We can keep it for reference but just thinking if vanilla customers know about the CNS backend as well, as they primarily interact with CSI only.

2. Are these dashboards specific to vanilla flavor and only for Grafana? We can name the folder appropriately, something like `grafana-dashboard`(if same across all flavors) or `vanilla-grafana-dashboard` ?

Reply:
For question 1, I think customers know about CNS.  We have many issues filed by customer that mention CNS volume.
For questions 2, yes,  let us name the folder as  "vanilla-grafana-dashboard".